docs/content/guides/operator/indexer-stack-setup.mdx

```diff
@@ -294,6 +294,86 @@ $ sui-indexer-alt-consistent-store run \
 	<ImportContent source="examples/prod-config/consistent-store.toml" mode="code" />
 </details>
 
+## GraphQL server setup
+
+GraphQL server instances read data from the indexer db, the consistent store and the archival store. The number of instances is determined by the required read throughput.
+
+#### Hardware requirements
+
+- **CPU:** 8 cores
+- **Memory:** 32GB
+
+### Run command
+
+GraphQL server supports two ways to access the archival store: self-hosted Bigtable instance, or a managed archival service that exposes a LedgerService gRPC interface. See [Archival Store](/concepts/archival-store) for more information.
+
+#### Run command using Bigtable
+
+Each GraphQL instance is run using the same command below:
+```sh
+GOOGLE_APPLICATION_CREDENTIALS=<PATH_TO_BT_CREDENTIALS_JSON> sui-indexer-alt-graphql rpc \
+    --config CONFIG_FILE \
+    --indexer-config INDEXER_CONFIG_FILE \
+    --bigtable-instance BIGTABLE_INSTANCE \
+    --consistent-store-url CONSISTENT_STORE_URL \
+    --database-url DATABASE_URL \
+    --fullnode-rpc-url FULLNODE_RPC_URL
+```
+
+#### Run command using LedgerService gRPC
+
+:::info
+
+If you use Sui Foundation's public good archival service at archive.testnet.sui.io and archive.mainnet.sui.io, you may experience performance issues which will be addressed before GA release.
+:::
+
+Each GraphQL instance is run using the same command below:
+```sh
+sui-indexer-alt-graphql rpc \
+    --config CONFIG_FILE \
+    --indexer-config INDEXER_CONFIG_FILE \
```

Comment:
Let's emphasise that multiple `--indexer-config`-s and indeed you should pass the config for every general-purpose indexer instance that you are running, so the GraphQL service is aware of all the pipelines it has access to. I see that it's mentioned in the table below and highlighted in the example, but maybe also show that there are multiple here too.